### ISSUE TITLE
Improved accessibility of HouseholdInfection_English 

### ISSUE THREAD

**Issue by krisnelson** [eleanormurray/COVID_19#30]
I marked various images as "decorative," gave alt text to some others, updated the reading order to make more sense, and gave the slide a title. I then generated a new, more accessible PDF from that. This does not impact any text/content nor the JPG. It should be much more accessible, but might need additional work based on feedback from screen-reader users. 

For more on doing this for other PowerPoint slides, see https://support.microsoft.com/en-us/office/make-your-powerpoint-presentations-accessible-to-people-with-disabilities-6f7772b2-2f33-4bd2-8ca7-dae3b2b3ef25?ui=en-us&rs=en-us&ad=us#PickTab=Windows 
[Household_English.pptx](https://github.com/eleanormurray/COVID_19/files/4422954/Household_English.pptx)
[HouseholdInfection_English.pdf](https://github.com/eleanormurray/COVID_19/files/4422957/HouseholdInfection_English.pdf)

**Comment by eleanormurray** [comment]
Thank you!!
